thumbor/handlers/__init__.py

```diff
@@ -281,7 +281,7 @@ def finish_request(self, context, result_from_storage=None):
             context.config.RESULT_STORAGE_STORES_UNSAFE or not context.request.unsafe)
 
         def inner(future):
-            results, content_type = future.result()
+            results, content_type = (future.result()[0], BaseEngine.get_mimetype(future.result()[0]))
```

Comment:
Ah, I see. 'auto' is not in `ALLOWED_FORMATS` so it's returning None. So, seems like we should drop the `if context.request.format` condition and just always set the MIME type. Seem logical?